test/test_torch.py

```diff
@@ -6984,6 +6984,59 @@ def test_bitwise_xor(self, device):
                          torch.bitwise_xor(torch.tensor([True, True, False], device=device),
                                            torch.tensor([False, True, False], device=device)))
 
+    @onlyOnCPUAndCUDA
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    @dtypes(*list(product(torch.testing.get_all_dtypes(include_complex=False),
+                          torch.testing.get_all_dtypes(include_complex=False))))
+    def test_heaviside(self, device, dtypes):
+        input_dtype = dtypes[0]
+        val_dtype = dtypes[1]
+
+        rng = np.random.default_rng()
+        input = np.array(rng.integers(-10, 10, size=10),
```

Comment:
Maybe add a few zeros to input? As written there's a chance it won't have any.

Reply:
`input[0] = input[3] = input[7] = 0`?

Reply:
Addressed!